src/explore-education-statistics-common/src/components/form/__tests__/FormComboBox.test.tsx

```diff
@@ -173,632 +179,552 @@ describe('FormComboBox', () => {
       expect(option2).toHaveAttribute('aria-selected', 'false');
       expect(option3).toHaveAttribute('aria-selected', 'true');
     });
-  });
-
-  describe('list box', () => {
-    test('pressing ArrowUp will cycle the selected item through the entire list', () => {
-      const { container, getByText, getByLabelText } = render(
-        <FormComboBox
-          id="test-combobox"
-          inputLabel="Choose option"
-          onInputChange={() => {}}
-          onSelect={() => {}}
-          options={['Option 1', 'Option 2', 'Option 3']}
-        />,
-      );
-
-      const input = getByLabelText('Choose option') as HTMLInputElement;
-
-      fireEvent.change(input, {
-        target: {
-          value: 'Test',
-        },
+    describe('list box', () => {
+      test('pressing ArrowDown will cycle the selected item through the entire list', () => {
+        const { container, getByText, getByLabelText } = render(
+          <FormComboBox
+            id="test-combobox"
+            inputLabel="Choose option"
+            onInputChange={noop}
+            onSelect={noop}
+            options={['Option 1', 'Option 2', 'Option 3']}
+          />,
+        );
+
+        const input = getByLabelText('Choose option') as HTMLInputElement;
+
+        fireEvent.change(input, {
+          target: { value: 'Test' },
+        });
+
+        const comboBox = container.querySelector(
+          '[role="combobox"]',
+        ) as HTMLElement;
+
+        const option1 = getByText('Option 1');
+        const option2 = getByText('Option 2');
+        const option3 = getByText('Option 3');
+
+        expect(option1).toHaveAttribute('aria-selected', 'false');
+        expect(option2).toHaveAttribute('aria-selected', 'false');
+        expect(option3).toHaveAttribute('aria-selected', 'false');
+
+        fireEvent.keyDown(comboBox, { key: 'ArrowDown' });
+
+        expect(option1).toHaveAttribute('aria-selected', 'true');
+        expect(option2).toHaveAttribute('aria-selected', 'false');
+        expect(option3).toHaveAttribute('aria-selected', 'false');
+
+        fireEvent.keyDown(comboBox, { key: 'ArrowDown' });
```

Comment:
This seems to be testing pressing arrow down to move through the list, but the test case originally was testing that you could press ArrowUp and go to to the end of the list and back up it. Why have we changed this?

Again, we have another test case further down called 'pressing ArrowDown will cycle the selected item through the entire list' that already tests going down the list. We should be testing going up the list!